### PR TITLE
changes label for total time limit

### PR DESCRIPTION
**Pull request by iqbalcodes6602**
# Description

Added the max time limit "max 6000" label to the total time limit.

image

Fixes: #7456 

# Comments

Add any comments for the reviewers (e.g., indicating the beginning of the revision, something where the reviewer needs to pay special attention, etc.). If  there are no extra comments this section can be deleted.

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [ ] If you are creating a feature, the new tests were added.
- [ ] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

**Review by pabo99 — APPROVED** [review]
@tvanessa is this translation okay? If so, for me this PR is approved

**Comment by iqbalcodes6602** [comment]
> @tvanessa is this translation okay? If so, for me this PR is approved

Hi unfortunately i don't know spanish or Portuguese, so i have used google translate, so it might not be exact, i can change it as you want

File: frontend/templates/en.lang [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ problemEditFormUpdateProblem = "Update problem"
 problemEditFormUpdateSolution = "Update solution"
 problemEditFormValidatorTimeLimit = "Validator time limit (ms)"
 problemEditFormValidatorType = "Output validator type"
-problemEditFormWallTimeLimit = "Total time limit (ms)"
+problemEditFormWallTimeLimit = "Total time limit (ms) (max 60000)"
```
Comment (heduenas) [inline frontend/templates/en.lang:1098]: can you please set it to "(max 60000 ms)"?

Comment (iqbalcodes6602) [reply frontend/templates/en.lang]: done as you said

**Pull request merged**